libbpf-cargo/src/build.rs

```diff
@@ -42,7 +42,10 @@ fn extract_version(output: &str) -> Result<&str> {
 }
 
 fn check_clang(debug: bool, clang: &Path, skip_version_checks: bool) -> Result<()> {
-    let output = Command::new(clang.as_os_str()).arg("--version").output()?;
+    let output = Command::new(clang.as_os_str())
+        .arg("--version")
+        .output()
+        .context("Failed to execute clang")?;
```

Comment:
if this fails, it's more of a "failed to find clang"?

Reply:
`.context` would bubble up the No such file or directory error

Reply:
Yeah, what @willfindlay  said